### ISSUE TITLE
The server does not start

### ISSUE THREAD

**Issue by C0RT3** [BoiseComputer/atlas-server-tools#31]
To begin, 

thank you for this superb work,

it is hard to follow an installation of an ATLAS server on the internet.

I followed the quick guide : `https://github.com/BoiseComputer/atlas-server-tools/wiki/Quick-Start`,

I did not receive an error return during the installation.

I only changed 2 things, the steam session name by atlas and the name of the game server.

this is what happens, when i try to start my server i get this:

`blabla:/home/atlas# atlasmanager start @all
Running command 'start' for instance 'main'
The server is starting...
The server is now running, and should be up within 10 minutes
Waiting up to 20 seconds before starting next instance ....................
Running command 'start' for instance 'a2'
The server is starting...
The server is now running, and should be up within 10 minutes
Waiting up to 20 seconds before starting next instance ....................
Running command 'start' for instance 'b1'
The server is starting...
The server is now running, and should be up within 10 minutes
Waiting up to 20 seconds before starting next instance ....................
Running command 'start' for instance 'b2'
The server is starting...
The server is now running, and should be up within 10 minutes
Waiting up to 20 seconds before starting next instance ....................
root@ns3123877:/home/atlas# atlasmanager status @all
Running command 'status' for instance 'main'
`

then when i order : **atlasmanager status @all** i get back

`blabla:/home/atlas# atlasmanager status @all
Running command 'status' for instance 'main'
 Server running:   No
 Redis Server running:   Yes
 Redis Server PID:   28486
 Server listening:   No
 Server build ID:   5267594
Running command 'status' for instance 'a2'
 Server running:   No
 Redis Server running:   Yes
 Redis Server PID:   28486
 Server listening:   No
 Server build ID:   5267594
Running command 'status' for instance 'b1'
 Server running:   No
 Redis Server running:   Yes
 Redis Server PID:   28486
 Server listening:   No
 Server build ID:   5267594
Running command 'status' for instance 'b2'
 Server running:   No
 Redis Server running:   Yes
 Redis Server PID:   28486
 Server listening:   No
 Server build ID:   5267594

`
as I don't see a LOG in the game, I don't know where to go.

I specify that I am under a Debian9 distribution.

Thank you in advance for your return.

**Comment by rksakhuja** [comment]
hi did you get anywhere with this problem?

also - thanks for your work with this @BoiseComputer
